src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -35,17 +35,23 @@ import {
 } from '@skyux/grids';
 
 import {
-  ListItemModel,
-  ListSortFieldSelectorModel,
   ListSearchModel,
   ListStateDispatcher,
   ListState
 } from '@skyux/list-builder/modules/list/state';
 
+import {
+  ListItemModel
+} from '@skyux/list-builder-common/state/items/item.model';
+
+import {
+  ListSortFieldSelectorModel
+} from '@skyux/list-builder-common/state/sort/field-selector.model';
```

Comment:
We can import from the barrel; exports are the only things that need the absolute path to the type.
```
import {
  getData,
  isObservable,
  ListItemModel,
  ListSortFieldSelectorModel
} from '@skyux/list-builder-common';
```

Reply:
There's a few other imports in the other files, too.

Reply:
And really, we're only writing those exports in this way for the skyux2 repo, to make sure that the specific files can be found.

Reply:
fixed